src/stores/manager/local/eth1_account.go

```diff
@@ -61,57 +59,5 @@ func NewEth1(ctx context.Context, specs *Eth1Specs, eth1Accounts database.ETH1Ac
 		return nil, err
 	}
 
-	err = InitDB(ctx, keyStore, eth1Accounts)
-	if err != nil {
-		return nil, err
-	}
-
-	return eth1.New(keyStore, eth1Accounts, logger), nil
-}
-
-func InitDB(ctx context.Context, keyStore keys.Store, db database.ETH1Accounts) error {
```

Comment:
IMO this code could be moved into a CLI cmd in oder to perform a DB synchonization on demand bases

Reply:
done: https://app.zenhub.com/workspaces/orchestrate-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/227